index/duke3d.toml

```diff
@@ -8,3 +8,4 @@ disabled = true
 
 [versions]
 "0.0.5" = {}
+"0.0.8" = {}
```

Comment:
Can you re-enable the world by deleting line 7? And drop "0.0.5" while you're at it since it was completely broken